index.d.ts

```diff
@@ -0,0 +1,50 @@
+export interface Editor {
+	id: string;
+	name: string;
+	bin: string;
+	isTerminalEditor: boolean;
+	paths: string[];
+	keywords: string[];
+}
+
+/**
+Get info about the default editor.
+
+The user is expected to have the `$EDITOR` environment variable set, and if not, a user-friendly error is thrown.
+
+@returns Info about the default editor.
+*/
+export default function(): Editor;
+
+/**
+Get info about a specific editor.
+
+@param editor - This can be pretty flexible. It matches against all the data it has.
+
```

Comment:
It looks weird to have an empty line here, even though the above belongs to the `editor` description.